pkgs/by-name/cp/cppman/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  groff
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "cppman";
+  version = "unstable-2024-02-15";
```

```suggestion
  version = "0-unstable-2024-02-15";
```
[See `pkgs/README.md` for more info about the version format](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)